Makefile.inc

```diff
@@ -1,7 +1,9 @@
 # Project name, used to name the binaries
 PKG_NAME := docker-machine
-GH_USER ?= docker
-GH_REPO ?= machine
+# Github infos
+GITHUB_INFO := $(shell git config --get remote.origin.url | sed -e 's/.*:\(.*\).git/\1/')
```

Comment:
I'm not sure what this is used for, but it seems a bit fishy to me.  Lots of users have different names for `origin`, `upstream`, and so on.  This also assumes that the remote is Github, which isn't guaranteed to be the case.